### ISSUE TITLE
Checking emails

### ISSUE THREAD

**Issue by kontovasBodleian** [fihristorg/fihrist-mss#132]
This is one of a series of issues I'll be posting about concerning things I notice as I put together my data entry system.

There are fewer issues with email addresses in our record files, but if those in the know could check the following that would be great.

<table>
    <tbody>
        <tr>
            <th>collection</th>
            <th>email(s)</th>
            <th>issue(s)</th>
        </tr>
        <tr>
            <td rowspan="2">arabic commentaries on the hippocratic aphorisms project</td>
            <td>peter.pormann@manchester.ac.uk</td>
            <td></td>
        </tr>
        <tr>
            <td>susan.thomas@unimelb.edu.au</td>
            <td></td>
        </tr>
        <tr>
            <td>british library</td>
            <td>apac-enquiries@bl.uk</td>
            <td></td>
        </tr>
        <tr>
            <td>cambridge university</td>
            <td>near_eastern@lib.cam.ac.uk</td>
            <td></td>
        </tr>
        <tr>
            <td>eton college windsor</td>
            <td></td>
            <td>no email</td>
        <tr>
        <tr>
            <td>hertford college (university of oxford)</td>
            <td>specialcollections.enquiries@bodleian.ox.ac.uk</td>
            <td></td>
        </tr>
        <tr>
            <td>jesus college (cambridge)</td>
            <td>old-library@jesus.cam.ac.uk</td>
            <td></td>
        </tr>
        <tr>
            <td>keble college (university of oxford)</td>
            <td>librarian@keble.ox.ac.uk</td>
            <td></td>
        </tr>
        <tr>
            <td>King's College Cambridge</td>
            <td>near_eastern@lib.cam.ac.uk</td>
            <td></td>
        </tr>
        <tr>
            <td>new college (university of oxford)</td>
            <td>naomi.vanloo@new.ox.ac.uk</td>
            <td></td>
        </tr>
        <tr>
            <td rowspan="2">oxford university</td>
            <td>peter.pormann@manchester.ac.uk</td>
            <td>is this correct for this collection?</td>
        </tr>
        <tr>
            <td>specialcollections.enquiries@bodleian.ox.ac.uk</td>
            <td></td>
        </tr>
        <tr>
            <td>queen's college cambridge</td>
            <td>library@queens.cam.ac.uk</td>
            <td></td>
        </tr>
        <tr>
            <td>royal asiatic society of great britain and ireland</td>
            <td>library@royalasiaticsociety.org</td>
            <td></td>
        </tr>
        <tr>
            <td>school of oriental and african studies</td>
            <td>libenquiry@soas.ac.uk</td>
            <td></td>
        </tr>
        <tr>
            <td>st antony's college (oxford)</td>
            <td>mastan.ebtehaj@sant.ox.ac.uk</td>
            <td></td>
        </tr>
        <tr>
            <td>the university of manchester</td>
            <td>uml.special-collections@manchester.ac.uk</td>
            <td></td>
        </tr>
        <tr>
            <td>trinity college cambridge</td>
            <td>wren.library@trin.cam.ac.uk</td>
            <td></td>
        </tr>
        <tr>
            <td>trinity college dublin, the university of dublin</td>
            <td>mscripts@tcd.ie</td>
            <td></td>
        </tr>
        <tr>
            <td>trinity hall (cambridge)</td>
            <td>library@trinhall.cam.ac.uk</td>
            <td></td>
        </tr>
        <tr>
            <td>university of birmingham</td>
            <td>special-collections@bham.ac.uk</td>
            <td></td>
        </tr>
        <tr>
            <td>university of st andrews</td>
            <td>speccoll@st-andrews.ac.uk</td>
            <td></td>
        </tr>
        <tr>
            <td>wadham college (university of oxford)</td>
            <td>persian.library@wadham.ox.ac.uk</td>
            <td></td>
        </tr>
        <tr>
            <td>wellcome trust</td>
            <td></td>
            <td>no email</td>
    </tbody>
</table>

**Comment by JakeBenson** [comment]
That's Peter's email, only I've no idea why he shows up as an Oxford contact. I alerted him to your other queries, so will see if he responds.

**Comment by kontovasBodleian** [comment]
Just modified the script I ran to check. Seems like the only one is MS_Huntingdon_359: https://www.fihrist.org.uk/catalog/manuscript_4049. This seems to be the source of the "The University of Manchester School of Arts, Languages and Cultures" publisher in the Oxford collection, too.

From the content of the ms, it seems like it actually makes sense for him to be affiliated with it, but I'm not sure if this is the right way to indicate that as I wasn't around for the project.

**Comment by JakeBenson** [comment]
Yes, that must be an error introduced during subsequent ehancements. His name should only display in relation to his project.

jb

From: Nicholas Kontovas (professional account) ***@***.***>
Date: Thursday, 20 April 2023 at 14:10
To: fihristorg/fihrist-mss ***@***.***>
Cc: Jake Benson ***@***.***>, Comment ***@***.***>
Subject: Re: [fihristorg/fihrist-mss] Checking emails (Issue #132)

Just modified the script I ran to check. Seems like the only one is MS_Huntingdon_359: https://www.fihrist.org.uk/catalog/manuscript_4049. This seems to be the source of the "The University of Manchester School of Arts, Languages and Cultures" publisher in the Oxford collection, too.

From the content of the ms, it seems like it actually makes sense for him to be affiliated with it, but I'm not sure if this is the right way to indicate that as I wasn't around for the project.

**Comment by kontovasBodleian** [comment]
Fixed the issue with the above record. Not sure how it got as messed up as it was, but effectively this was an inferior (and misspelled) record for <a href="https://www.fihrist.org.uk/catalog/manuscript_1002">MS. Huntington 359</a> which showed up as being part of the Arabic Aphorisms project despite being in the Oxford folder and on an Oxford ms! I've removed the inferior record, ported additional information from there to the superior one, and moved the work entry created for the inferior record to <a href="https://github.com/fihristorg/fihrist-mss/blob/master/authority/works_deletions.xml">`works_deletions.xml`</a>.

Since this was the only real problem with emails, I'm marking this issue closed.
